pkg/services/oracle/filter.go

```diff
@@ -3,13 +3,445 @@ package oracle
 import (
 	"encoding/json"
 	"errors"
+	"sort"
+	"strconv"
+	"strings"
 	"unicode/utf8"
 
-	"github.com/PaesslerAG/jsonpath"
 	"github.com/nspcc-dev/neo-go/pkg/core/state"
 	"github.com/nspcc-dev/neo-go/pkg/core/transaction"
 )
 
+type (
+	// pathTokenType represents single JSONPath token.
+	pathTokenType byte
+
+	// pathParser combines JSONPath and a position to start parsing from.
+	pathParser struct {
```

Comment:
Can we move JsonPath implementation to a separate package?

Reply:
Moved.